terminus

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * If we're running from phar load the phar autoload file.
+ */
+$pharPath = \Phar::running(true);
+if ($pharPath) {
+    require_once "$pharPath/vendor/autoload.php";
+} else {
+    if (file_exists(__DIR__.'/vendor/autoload.php')) {
+        require_once __DIR__.'/vendor/autoload.php';
+    } elseif (file_exists(__DIR__.'/../../autoload.php')) {
+        require_once __DIR__ . '/../../autoload.php';
+    }
+}
+
+$discovery = new \Consolidation\AnnotatedCommand\CommandFileDiscovery();
+$discovery->setSearchPattern('*Command.php');
+$commandClasses = $discovery->discover('php/Terminus/AnnotatedCommands', '\Terminus\AnnotatedCommands');
```

Comment:
The discovery class has a bunch of searching options, but at its simplest, just give it a path and a namespace where command files can be found.  This actually needs to be adjusted to use __DIR__ or $pharPath, so that it works once built into a phar.  Haven't done that yet.